charts/dapr/crds/subscription.yaml

```diff
@@ -21,7 +21,7 @@ spec:
       - v1
       - v2alpha1
   versions:
-  - name: v1alpha1
+  - name: v1
```

Comment:
For Subscriptions, we need to add `v1` with the same schema as `v2alpha1` and leave `v1alpha` unchanged.

Reply:
And we also need to keep `v2alpha1` around because that is also in use.